src/components/clusters/wizards/form/CheckboxField.test.tsx

```diff
@@ -6,12 +6,14 @@ import { waitFor } from '@testing-library/react';
 import { acknowledgePrerequisites } from '~/common/validators';
 import { checkAccessibility, render, screen } from '~/testUtils';
 
-import { CheckboxField } from './CheckboxField';
+import { CheckboxField, CheckboxFieldProps } from './CheckboxField';
+
+type CheckboxFieldOptionalProps = Omit<CheckboxFieldProps, 'name'>;
```

Comment:
This type's purpose is to help add optional properties to the rendered component. 
All of it's props are optional except for `name` which is required. In the current implementation adding it to the `additionalProps` every time the component is being tested with another optional prop seem redundant, so I omitted it via this type to add to the `prepareComponent` function an option to add optional props to the rendered component